src/components/error-message/index.md

```diff
@@ -59,6 +59,20 @@ There are 2 ways to use the error message component. You can use HTML or, if you
 
 {{ example({ group: "components", item: "error-message", example: "label", html: true, nunjucks: true, open: false, size: "s" }) }}
 
+### Match up error messages to labels
+
+Error messages should directly include language from the question or fieldset label. This helps match up the error message with the relevant form field.
+
+Here are some examples of label and error message pairs.
+
+Example 1:
```

Comment:
Prettier has flagged this (missing new line) just to make sure Markdown always renders correctly

```suggestion
Example 1:

```